pyproject.toml

```diff
@@ -1,13 +1,19 @@
 [project]
 name = "pdal_ign_macro"
 dynamic = ["version"]
+readme = "README.md"
+description = "Macros and pipelines for Pdal based on the https://github.com/IGNF/pdal-ign-plugin plugin"
```

Comment:
Comme c'est la description du package python, j'aurai mis un truc qui fait la difference entre le python et le c++, comme ça :

"Python examples to use C++ PDAL plugin https://github.com/IGNF/pdal-ign-plugin"

Enfin c'est qu'une suggestion, c'est comme tu préfères.